src/components/tools/Partials/ChangeLocation.svelte

```diff
@@ -70,7 +70,7 @@
 
   async function updateBoundary(e) {
     if (!e.detail) return;
-    const { lng, lat } = currentLoc.center;
+    const [lng, lat] = currentLoc.center || [];
```

Comment:
Maybe instead of `[]` this could be the default location center coords.

Reply:
Sounds good.

Reply:
@bkg   the `distance_to` param is supported for station datasets, is it also supported for boundaries?  

Station example - https://api.cal-adapt.org/api/hadisdstations/?distance_to=POINT(-124+40)
this works, a `distance` property is added to `properties` for each feature

Boundary example -https://api.cal-adapt.org/api/place/?srs=4326&precision=4&distance_to=POINT(-120.7813%2038.7188)
 this does not add a `distance` property to each feature

Reply:
I followed up on Trello, but currently no, `distance_to` is only available for HadISD stations. I suppose it's only a problem for any sparse boundary layers like places since a simple intersect will return a result for all the others.